src/KeyExchange/DiffieHellmanGroup.php

```diff
@@ -0,0 +1,219 @@
+<?php
+
+namespace Amp\Ssh\KeyExchange;
+
+
+use function Amp\call;
+use Amp\Promise;
+use Amp\Ssh\Message\KeyExchangeCurveInit;
+use Amp\Ssh\Message\KeyExchangeCurveReply;
+use Amp\Ssh\Message\Message;
+use Amp\Ssh\Transport\BinaryPacketHandler;
+
+/**
+ * @internal
+ */
+class DiffieHellmanGroup implements KeyExchange
```

Comment:
add final here, i don't think there is a need to inherit this

Reply:
ok